sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/tests/OpenTelemetry.Exporter.AzureMonitor.IntegrationTests/OpenTelemetry.Exporter.AzureMonitor.Integration.Tests.csproj

```diff
@@ -0,0 +1,39 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+  <!-- https://docs.microsoft.com/aspnet/core/test/integration-tests -->
+  
+  <PropertyGroup>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.Testing" VersionOverride="2.1.1" />
+
+    <PackageReference Include="OpenTelemetry.Instrumentation.AspNetCore" VersionOverride="0.6.0-beta.1" />
+    <PackageReference Include="OpenTelemetry.Extensions.Hosting" VersionOverride="0.6.0-beta.1" />
```

Comment:
Curious, why `VersionOverride` instead of `Version`?

Reply:
Whenever there's a conflict with the packagereferences brought it by Azure SDK, using `VersionOverride` always fixes it.
I defaulted to using it while just trying to get everything to pass in the build pipeline. 

I'll do one more check now and see which can and cannot be removed